package.json

```diff
@@ -39,6 +39,8 @@
     "coffeelint": "1.15.0",
     "jade2commonjs": "0.1.0",
     "mocha": "2.4.5",
+    "nodemon": "^1.9.1",
+    "npm-run-all": "^1.7.0",
```

Comment:
Cozy has guidelines to use strict dependencies (no `^`). You may add `save-exact=true` to your npmrc.